src/main/java/jarvis/logic/parser/ParserUtil.java

```diff
@@ -195,4 +195,17 @@ public static Set<Index> parseStudentIndexes(Collection<String> studentIndexes)
         return studentIndexSet;
     }
 
+    /**
+     * Parses a {@code String participation} into an {@code int} indicating the participation amount.
+     *
+     * @throws ParseException if the given {@code participation} is invalid.
+     */
+    public static int parseParticipation(String participation) throws ParseException {
+        String trimmedParticipation = participation.trim();
+        try {
+            return Integer.parseInt(trimmedParticipation);
```

Comment:
To remain consistent with `StudioParticipation`, I think this should check that the participation value is an integer between 0 and 500 inclusive and throw an exception otherwise.